en/contact/index.md

```diff
@@ -3,7 +3,13 @@ Title: Contacts us information
 layout: contact
 lang: EN
 ref: contact
+street: 1100 Notre-Dame Ouest, Montreal, QC, H3C 1K3
```

Comment:
The following info is redundant on each page and could also be used else where in the site please move it to _config.yml